source/process/training.rst

```diff
@@ -368,11 +368,13 @@ Dev Mana
 
 A specific type of mana for developers similar to "points" or "jelly beans" in an Agile/Scrum methodology. On average, full time Mattermost developers each complete tickets adding up to approximately 28 mana per week. A "small" item is 2 mana, a "medium" is 4, a "large" is 8 and any project bigger needs to be broken down into smaller tickets.
 
-Expert Mode
+Expert Mode (and Crimson Force Field)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 When documentation or on-screen text is written for someone with considerable knowledge or expertise, instead of being designed for a new learner. In general, try to state things simply rather than speaking to just the "experts" reading the the text. 
 
+If something is extremely difficult to understand, and yet still justified in the mind of the designer, we call it "Crimson Force Field" as a way to illustrate how arbitrary and confusing the term appears to the person sharing feedback. 
```

Comment:
Is Crimson Force Field explained elsewhere? Maybe explain where the metaphor comes from, or else "Crimson Force Field" here feels a little "Crimson Force Field"-y?